scrunch/cubes.py

```diff
@@ -0,0 +1,12 @@
+from pycrunch.cubes import fetch_cube, count
+from cr.cube.crunch_cube import CrunchCube
+
+
+def crtabs(dataset, variables):
+    """Return CrunchCube representation of crosstab.
+
+    :param dataset: Dataset shoji object
+    :param variables: List of variable urls, names or aliases
```

Comment:
Note about this `variables`, chances are high this will be a `scrunch.Variable`, I'd recommend to also expect that and if they're such instances, convert them to that.

If you can add a test would help.